### ISSUE TITLE
"P3S Bright Fire Marker" or is it "P3S Bright Fire Marker and Fledgling Flights" ?

### ISSUE THREAD

**Issue by woodshop2300** [quisquous/cactbot#4044]
[raidboss]
On the "Cactbot Config" tab P3S for what it calls "P3S Bright Fire Marker" overriding the trigger alert text does not appear to work.
I think i might have a hint as to why. 

You can normally click the "View Trigger Source" link to popup a window directly to the trigger source.
For what the "Cactbot Config" lists as "P3S Bright Fire Marker" this does not work, my guess is because the id: is actually "P3S Bright Fire Marker and Fledgling Flights" in the .js

I'm guessing there is some kind of implicit name mapping going on somewhere as part of the override system and things are failing to match up.

Then since were on this section of the config, the "P3S Bright Fire Marker" section only has settings for Bright Fire Marker, the Fledgling Flights parts are missing settings in the P3S section completely.

TLDR: I think the UI Config page section for P3S and the triggers got de-synced
![STKolxx](https://user-images.githubusercontent.com/16126591/150081879-ccfb7cad-bf4a-4a9f-b460-8b8b710d354d.png)
.

**Comment by valarnin** [comment]
Probably not actually a bug. This commit https://github.com/quisquous/cactbot/commit/a41e26dd2c59ffcbfd4237f056fdb8f45d1c187e changed the trigger name.

You're pointed to the github.io link, which will use the latest version of triggers and timelines, however the built-in `Cactbot Config` tab always uses the information built into the plugin release DLL that ACT is running. This is one of those risks with using the github.io link.

**Comment by woodshop2300** [comment]
Your referring to using https://quisquous.github.io/cactbot/ui/raidboss/raidboss.html vs the on disk version in AppData/Roaming/Advanced%20Combat%20Tracker/Plugins/cactbot-0.24.14/cactbot/ui/raidboss/raidboss.html i think?

Unless switching that URL back and forth and reloading ACT isn't enough to switch between the two i am currently using the on disk version. Switching to the on disk version is normally what i try first since things can land before versions get pushed out etc. 
![ilGweHa](https://user-images.githubusercontent.com/16126591/150190866-6d57a5d8-4859-469f-9c5e-b019137117da.png)

before i made this i also went and tried to hunt down the triggers that we on the disk, i found the data folders like normal just fine, but, the 06-ew stuff was miss etc.. then pushed me to do some cleaning and i whiped out overlap and cactbot and pulled them down fresh.. 

Based on what it pulled down it appears that at some version a lot of this went remote since all the timeline and trigger data is MIA on disk there days.. (or they moved it outside the cactbot folder).
![RlsDRHK](https://user-images.githubusercontent.com/16126591/150192019-f9616dc3-260e-4ed5-8972-0d590816a5d2.png)

**Comment by valarnin** [comment]
Check `\Plugins\cactbot-0.24.14\cactbot\ui\common\raidboss_data.bundle.js`, search for `P3S Bright` (it's on line 162242 for me).

![image](https://user-images.githubusercontent.com/6119598/150193484-cba87c8e-4972-46b3-9556-d6378c4da526.png)

**Comment by woodshop2300** [comment]
Ahh ok, they are concatenating it all together now on release.. thats why i couldn't find it.. 
Its there, cool.
Welp only 1 way to test, and thats to wait 7 hours.

**Comment by woodshop2300** [comment]
After some issues getting tts working again things worked. Cheers, no bug.
